playground/backend/internal/setup_tools/compile_builder/compile_builder_setuper.go

```diff
@@ -28,10 +28,16 @@ func Setup(filePath, filesFolderPath string, sdk pb.Sdk, executorConfig *environ
 	if err != nil {
 		return nil, err
 	}
+	prep, err := utils.GetPreparators(sdk, filePath)
+	if err != nil {
+		return nil, err
+	}
 
 	compileBuilder := executors.NewExecutorBuilder().
 		WithValidator().
 		WithSdkValidators(val).
+		WithPreparator().
+		WithSdkPreparators(prep).
```

Comment:
For Python we don't have compile step. Would preparators work in this case?

Reply:
`Prepare step` works using `executor` from `compileBuilder`:
```
executor := compileBuilder.Build()
...
// validate step
...
// prepare step
prepareFunc := executor.Prepare()
go prepareFunc(successChannel, errorChannel)
...
// compile step
...
```

So it seems that it should work and compile step doesn't have any effect on Prepare step.

Reply:
ok, the "compile" word in the name of compileBuilder confuses me